src/app/core/services/logging.service.ts

```diff
@@ -53,7 +53,7 @@ export class LoggingService {
    * of Sessions if necessary.
    * @param sessionCount The number of Session Logs to preserve in the cache
    */
-  getTrimmedLogCache(currentLog: string, sessionCount: number): string {
+  private getTrimmedLogCache(currentLog: string, sessionCount: number): string {
     const sessionLogSeparator: string = '\n'.repeat(2); // More new-lines added for clarity.
```

Comment:
Just a suggestion:

Should we make this `sessionLogSeparator` a `public readonly` class field instead of local variable inside `getTrimmedLogCache`?

By making it a class field, we can then use it in the testing file as
`const sessionSeparator = loggingService.SESSION_LOG_SEPARATOR;`
In my opinion, this will make the tests more extensible and general as changes to the session log separator in the implementation will no longer break the tests.